src/dcat-us/compile-dcat-feed.ts

```diff
@@ -25,6 +22,32 @@ function removeUninterpolatedDistributions(distributions: any[]) {
   return distributions.filter((distro) => !(typeof distro === 'string' && distro.match(/{{.+}}/)?.length));
 }
 
+function generateDcatItem(feedTemplate, feedTemplateTransforms, geojsonFeature) {
```

Comment:
this is typescript, but you have not given the parameters types, and the function doesn't have a return type.

Reply:
Yes, missed the types. I had added them. However. let me know if I need to narrow down my types to be even more concrete.